extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/VarBinaryChunkInputStreamGenerator.java

```diff
@@ -328,15 +337,14 @@ public int drainTo(final OutputStream outputStream) throws IOException {
                     if (offset + length > serializedData.length) {
                         throw new IllegalStateException("not enough data was serialized to parse this element");
                     }
-                    chunk.set(ii, mapper.constructFrom(serializedData, offset, length));
+                    chunk.set(outOffset + ii, mapper.constructFrom(serializedData, offset, length));
```

Comment:
This notably differs from the validity-buffer-backed code in other generators. Should it converge?